src/main/kotlin/com/senacor/lpt/service/creditapplication/decision/CreditDecisionService.kt

```diff
@@ -0,0 +1,23 @@
+package com.senacor.lpt.service.creditapplication.decision
+
+import com.senacor.lpt.service.creditapplication.CreditDecisionType
+import com.senacor.lpt.service.creditapplication.CreditDecisionType.*
+import org.springframework.stereotype.Service
+import reactor.core.publisher.Mono
+import java.math.BigDecimal
+
+@Service
+class CreditDecisionService {
+
+    val minimalNetIncome = BigDecimal(1000);
+
+    fun decideOnCredit(monthlyNetIncome: BigDecimal,
+                       monthlyExpenses: BigDecimal
+    ): Mono<CreditDecisionType> {
+        return if (monthlyNetIncome.minus(monthlyExpenses) > minimalNetIncome) {
+            Mono.just(APPROVED)
+        } else {
+            Mono.just(DENIED)
+        }
+    }
```

Comment:
Hier kannst du auch schön abkürzen:
```
... ) = if (monthlyNetIncome.minus(monthlyExpenses) > minimalNetIncome) {
        Mono.just(APPROVED)
    } else {
        Mono.just(DENIED)
    }
```

Reply:
bzw in Kotlin mit when statt if, weil "schöner" und "kürzer" und "more kotlin" :D